source/manual/covid-19-services.html.md

```diff
@@ -164,9 +164,9 @@ settings and credentials for the backing service automatically.
 
 We have two AWS accounts (staging and production) for CDN and data
 storage. They contain DynamoDB, IAM and CloudFront resources,
-which were provisioned by Terraform in the [tech-ops-private
-repository](techops-repo). Members of the GOV.UK Coronavirus Services
-team have access.
+which were provisioned by Terraform in the [alphagov/covid-engineering
+repository](covid-engineering-repo). Members of the GOV.UK Coronavirus
```

Comment:
[alphagov/covid-engineering repository](https://github.com/alphagov/covid-engineering)

Reply:
Thanks, turns out the syntax was entirely wrong!